### PR TITLE
Add complete BCD for Window Management API

### PR DESCRIPTION
**Pull request by chrisdavidmills**
#### Summary

The [Window Management API](https://w3c.github.io/window-management/) was implemented in Chrome 100 (see [ChromeStatus entry](https://chromestatus.com/feature/5252960583942144)); this PR provides complete BCD for it (some was available already).

### CODE REVIEW

File: api/ScreenDetailed.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
       },
       "availLeft": {
         "__compat": {
-          "spec_url": "https://w3c.github.io/window-management/#ref-for-dom-screendetailed-availleft",
```
Comment (Elchi3) [inline api/ScreenDetailed.json:37]: Why is this change necessary? 

@sideshowbarker has opinions about where to link in the specs. I think it has to do with where the MDN ribbon appears.

**Review by Elchi3 — CHANGES_REQUESTED** [review]
LGTM with two comments.

File: api/ScreenDetailed.json (hunk 1 above)
Comment (sideshowbarker) [reply api/ScreenDetailed.json:37]: The practice followed by the majority of the existing spec URLs is to use the URL for the developer part of the spec, if any. In this case, that’s https://w3c.github.io/window-management/#ref-for-dom-screendetailed-availleft

And yeah the effect of that as far as the rendering of the published specs goes is that it causes the **MDN** annotation in the spec to be shown next the developer part of the spec (rather than the implementor part).

File: api/Permissions.json [hunk 2]
```diff
@@ -891,4 +927,4 @@
       }
     }
   }
-}
+}
```
[the remark this answers is not in the document]
Comment (chrisdavidmills) [reply api/Permissions.json]: Oh wow, I honestly never knew this was a thing to be concerned about! I have removed the odd newline here and there because I thought they were unnecessary.

I've updated my config so VSCode adds newlines to the end of all files on save, and added the missing ones into this PR. Sorted.

**Review by Elchi3 — APPROVED** [review]
Perfect, thank you Chris! 👍
